firebase-database/firebase-database.gradle

```diff
@@ -86,20 +83,23 @@ dependencies {
     androidTestImplementation "com.android.support:support-annotations:$supportAnnotationsVersion"
     androidTestImplementation 'com.android.support.test:rules:1.0.2'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.google.truth:truth:0.39'
+    androidTestImplementation 'com.google.truth:truth:0.42'
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-core:2.9.6'
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.9.6'
     androidTestImplementation 'junit:junit:4.12'
-    androidTestImplementation 'net.java.quickcheck:quickcheck:0.6'
+    androidTestImplementation 'net.java:quickcheck:0.6'
+    androidTestAnnotationProcessor 'net.java:quickcheck-src-generator:0.6'
+    androidTestAnnotationProcessor 'net.java.quickcheck:quickcheck-src-generator:0.6'
```

Comment:
Ditto

Reply:
Same as above. But this time, somehow Gradle needs 2 of these `annotationProcessor` to be devlared explicitly. Or else it will fail. I'm still wondering why, I think there's an inconsistency with `quickcheck` dependency in using the annotationProcessor.

Reply:
After some times of experimenting, I guess it's not failing. But somehow, often times the CI system will treat this as error and failing the build. What do you think @ashwinraghav ?

Reply:
Backlog references: 
- Failed in `check` process. https://k8s-gubernator.appspot.com/build/android-ci/pr-logs/pull/firebase_firebase-android-sdk/73/check/1052633649951805440/
- This build requires annotationProcessor for butterknife. But after `retest` works again.
https://k8s-gubernator.appspot.com/build/android-ci/pr-logs/pull/firebase_firebase-android-sdk/73/connected-check/1056156782134562817/